assets/js/components/Banner/LearnMoreLink.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Link from '../Link';
+
+export default function LearnMoreLink( {
+	href,
+	className,
+	label = __( 'Learn more', 'google-site-kit' ),
+} ) {
+	return (
+		<Link href={ href } external className={ className }>
```

Comment:
All properties with no value should be moved to the end of the element:
```suggestion
		<Link href={ href } className={ className } external>
```

The same applies to other occurrences in this PR.